### PR TITLE
[Docs] update Japanese translation of adc_driver.md

### PR DESCRIPTION
**Pull request by umi-umi**
## Description

update Japanese translation of docs/adc_driver.md.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

File: docs/ja/adc_driver.md [hunk 1]
```diff
@@ -146,10 +152,10 @@ SRC += analog.c
...
+|---------------------|------|--------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|`ADC_CIRCULAR_BUFFER`|`bool`|`false`                                           | `true` の場合、この実装は循環バッファを使います。                                                                                                                                                                           |
+|`ADC_NUM_CHANNELS`   |`int` |`1`                                               | ADC 動作の一部としてスキャンされるチャンネル数を設定します。現在の実装は `1` のみをサポートします。                                                                                                                         |
+|`ADC_BUFFER_DEPTH`   |`int` |`2`                                               | 各結果の深さを設定します。デフォルトでは10ビットの結果しか取得できないため、これを2バイトに設定して1つの値を含めることができます8ビット以下の結果を選択した場合は、これを 1 に設定できます。                                |
```
Comment (s-show) [inline docs/ja/adc_driver.md:159]: 句読点が抜けているようです。
```suggestion
|`ADC_BUFFER_DEPTH`   |`int` |`2`                                               | 各結果の深さを設定します。デフォルトでは10ビットの結果しか取得できないため、これを2バイトに設定して1つの値を含めることができます。8ビット以下の結果を選択した場合は、これを 1 に設定できます。                                |
```

Comment (umi-umi) [reply docs/ja/adc_driver.md]: ご指摘ありがとうございます！

**Review by s-show — APPROVED** [review]

**Review by peepeetee — APPROVED** [review]

**Comment by zvecr** [comment]
Closing as this would have gone stale multiple times by now.

**Pull request closed without merging**